### PR TITLE
Add Thumbor plugins new structure

### PR DESCRIPTION
**Pull request by guilhermef**
**What kind of change does this PR introduce?**

- [ ] Add a project
- [X] Update a project
- [ ] Remove a project
- [ ] Add or update a category
- [ ] Change configuration
- [ ] Documentation
- [ ] Other, please describe:

**Description:**

Add Thumbor Plugins new structure
**Checklist:**

- [X] I have read the [CONTRIBUTING](https://github.com/best-of-lists/best-of/blob/main/CONTRIBUTING.md) guidelines.
- [X] I have not modified the `README.md` file. Projects are only supposed to be added or updated within the `projects.yaml` file since the `README.md` file is automatically generated.

### CODE REVIEW

File: projects.yaml [hunk 1]
```diff
@@ -262,9 +262,39 @@ projects:
     category: filters
 
     # Optimizers
-  - name: thumbor-plugins
+  - name: thumbor-plugins-gifv
```
Comment (scorphus) [inline projects.yaml:265]:
```suggestion
  - name: coover-thumbor-plugins
    github_id: thumbor/thumbor-plugins
    pypi_id: coover-thumbor-plugins
    labels: []
    category: optimizers
  - name: thumbor-plugins-gifv
```

**Review by scorphus — APPROVED** [review]
Nice one!

Nit: is [coover-thumbor-plugins](https://pypi.org/project/coover-thumbor-plugins/) missing from that bunch perhaps?

File: projects.yaml (hunk 1 above)
Comment (guilhermef) [reply projects.yaml:265]: We can also add this cover project, but it's not part of the thumbor-plugins GitHub repo.

Comment (scorphus) [reply projects.yaml:265]: True, I was tricked by the repo name.

**Pull request merged**